kotlinpoet/src/main/java/com/squareup/kotlinpoet/KmSpecs.kt

```diff
@@ -289,7 +299,17 @@ private fun ImmutableKmProperty.toPropertySpec(
         mutable(false)
       }
       if (isDelegated) {
-        delegate("") // Placeholder
+        // Placeholders for these are tricky
+        addKdoc("Note: delegation is ABI stub only and not guaranteed to match source code.")
```

Comment:
I think it's useful just since this is a case where we're clearly fabricating behavior for the sake of compilation and not actually matching source behavior. Similar to the `TODO("Stub!")` holders, but the only way we can express that here is a comment :/

Reply:
Okay, let's leave it